src/lib/i18n/locales/languages.json

```diff
@@ -114,5 +114,9 @@
 	{
 		"code": "dg-DG",
 		"title": "Doge 🐶"
+	},
+	{
+		"code": "hr-HR",
+      	"title": "Croatian"
```

Comment:
`"title": "Croatian"` needs another... 8 spaces in front of it to be aligned with `"code": "hr-HR",`